histomicstk/FeatureExtraction.py

```diff
@@ -115,6 +112,24 @@ def FeatureExtraction(Label, I, K=128, Fs=6, Delta=8):
         MinorAxisLength = np.append(MinorAxisLength, region.minor_axis_length)
         Extent = np.append(Extent, region.extent)
         Solidity = np.append(Solidity, region.solidity)
+        # get bounds of dilated nucleus
+        # region.bbox : min_row, min_col, max_row, max_col
+        # bounds : min_row, max_row, min_col, max_col
+        bounds = GetBounds(region.bbox, Delta, size_x)
+        # grab nucleus mask
+        Nucleus = (
+            Label[bounds[0]:bounds[1], bounds[2]:bounds[3]] == region.label
+        ).astype(np.int)
+        # get Bounds
+        Bounds = np.argwhere(
+            find_boundaries(Nucleus, mode="inner").astype(np.uint8) == 1
+        )
+        # calculate FSDs
+        FSD = FSDs(
+            Bounds[:, 0], Bounds[:, 1],
+            K, Interval
+        )
+        FSDGroup = np.append(FSDGroup, FSD)
 
     # add columns to dataframe
     df['X'] = CentroidX
```

Comment:
My bad! I now see that you have two separate features/columns for the x-coordinate (CentroidX) and y-coordinate (CentroidY). All good on this :)